torchvision/models/detection/backbone_utils.py

```diff
@@ -40,10 +40,14 @@ def __init__(self, backbone, return_layers, in_channels_list, out_channels):
         self.out_channels = out_channels
 
 
-def resnet_fpn_backbone(backbone_name, pretrained):
+def resnet_fpn_backbone(backbone_name, pretrained, export_onnx=False):
+    if export_onnx:
```

Comment:
@fmassa I really don't want to do this, but can't see a better way.

I want to avoid sending an arg like export_onnx to specify that we are exporting the model to ONNX, but the issue is that FrozenBatchNorm2d is not traceable;
FrozenBatchNorm2d has its forward method as a @torch.jit.script_method, and is not traceable.
I get the error :
RuntimeError: Tried to trace <__torch__.torchvision.ops.misc.FrozenBatchNorm2d object at ...> but it is not part of the active trace. Modules that are called during a trace must be registered as submodules of the thing being traced.

I added FrozenBatchNorm2d_ONNX without the script_method, and the issue is resolved, but duplicating FrozenBatchNorm2d is not great.

Any ideas?

Reply:
@eellison This is probably legacy from the old interface, but maybe the `@torch.jit.script_method` was not required for `forward`?

Anyway, I'm ok removing the decorator, and even maybe making the whole class a pure `nn.Module` (although this will have impact on memory and runtime performance)